MichelleYu/home.html

```diff
@@ -0,0 +1,103 @@
+                    The acting is great, especially Leo Dicaprio and Marion Cottilard's, it's a big part of why this movie is so good. They really act like it's happening in real life. The cast fit the characters to a tee. <br><br>This thriller certainly kept me on the edge of my seat. 8.5/10
+                </div>
+            </div>
+
+            <div class="fullwrap">
+                <img src="love_and_thunder.jpg" alt="Thor: Love and Thunder" width="500" height="700">
+                <div class="fullcap">
+                    Thor: Love and Thunder<br><br>A fun ride<br><br>Going into this movie, I had never watched a Marvel movie so do please take this review with a grain of salt. Thor: Love and Thunder was an action-packed, emotional romantic journey and quite comedic too. I loved watching Thor's character development since it honestly was a series of humbling events lol. The special effects and film choices were nicely done.<br><br>I may have not been able to understand any of the references but I can confidently say that Love and Thunder delivered wholesome entertainment and a visually grandiose theatre experience. Definitely recommend. 8/10
+                </div>
+            </div>
+        </div>
+        <div class="double">
+            <div class="fullwrap">
+                <img src="ride your wave.jpg" alt="Ride Your Wave" width="500" height="700">
+                <div class="fullcap">
+                    Ride Your Wave<br><br>Emotional and Bittersweet<br><br>At the beginning, this anime was really sweet, I had no idea I would have been sobbing straight through it and even afterwards. I am in love with the storyline and the characters within it. Hinako and Minato, the main characters, were so endearing and watching Hinako's character development and struggle with greif was really moving. The animation and color choices were unique.<br><br>Though the story was sad, but it is still very wholesome and nice. I think it is a perfect summer holiday activity. 8/10
+                </div>
+            </div>
+
+            <div class="fullwrap">
+                <img src="weathering with you.jpg" alt="Weathering With You" width="500" height="700">
+                <div class="fullcap">
+                    Weathering With You<br><br>Pure delight and tearfully gorgeous<br><br>This story follows a runaway teenaged boy who meets a sunshine girl in Tokyo. It dwells on the theory of fate, where one meets his other half through supernatural means and grit. This is the film's main theme, where the environment is used to appeal than solidify as a breathtaking experience for most audiences. Although instilled with fantasy and Shinto mythology, there is still a hint of reality. References of global warming, child protective services, and struggles in a capitalist society are woven in masterfully.<br><br>Weathering with You is a beautifully animated film about young, spontaneous love that transcends mother nature. 8.5/10
```

Comment:
You can put the <br> tags on another line and move the following text on another line.